sale_by_packaging/models/product_template.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    sell_only_by_packaging = fields.Boolean(
+        string="Only sell by packaging",
+        default=False,
+        help="Restrict the usage of this product on sale order lines without "
+        "packaging defined",
+    )
+
+    @api.constrains("sell_only_by_packaging", "sale_ok", "packaging_ids")
```

Comment:
do you need a "packaging_ids" field here?

Reply:
Not needed indeed, good catch